grammars/scss.cson

```diff
@@ -329,48 +329,92 @@
       }
     ]
   'at_rule_keyframes':
-    'patterns': [
-      {
-        'begin': '^\\s*((@)(-[\\w-]*-)?keyframes\\b)\\s*([\\w-]*)'
-        'captures':
-          '1':
-            'name': 'keyword.control.at-rule.keyframes.scss'
-          '2':
-            'name': 'punctuation.definition.keyword.scss'
-          '3':
-            'name': 'punctuation.definition.keyword.scss'
-          '4':
-            'name': 'entity.name.function.scss'
-        'comment': 'Keyframes with Attributes'
-        'end': '(?<=})(?:\\s*$)?'
-        'name': 'meta.at-rule.keyframes.scss'
-        'patterns': [
-          {
-            'begin': '{'
-            'end': '}'
-            'beginCaptures':
-              '0':
-                'name': 'punctuation.section.keyframes.begin.scss'
-            'endCaptures':
-              '0':
-                'name': 'punctuation.section.keyframes.end.scss'
-            'name': 'meta.keyframes.scss'
-            'patterns': [
-              {
-                'match': '(\\b(\\d+%|from\\b|to\\b))'
-                'name': 'entity.other.attribute-name.scss'
-              }
-              {
-                'include': '#interpolation'
-              }
-              {
-                'include': '#property_list'
-              }
-            ]
-          }
-        ]
-      }
-    ]
+    {
+      'begin': '(@)(?:-(?:webkit|moz)-)?keyframes\\b'
+      'beginCaptures':
+        '0':
+          'name': 'keyword.control.at-rule.keyframes.scss'
+        '1':
+          'name': 'punctuation.definition.keyword.scss'
+      'end': '(?<=})'
+      'name': 'meta.at-rule.keyframes.scss'
+      'patterns': [
+        {
+          'match': '(?<=keyframes[ \\t])(?:[_A-Za-z][-\\w]|-[_A-Za-z])[-\\w]*'
```

Comment:
I'm not liking this lookbehind because it doesn't take into account multiple spaces.